Blimp/Parameters.h

```diff
@@ -60,12 +57,6 @@ class Parameters
         // scheduler object (for debugging)
         k_param_scheduler,
 
-        // relay object
-        k_param_relay,
```

Comment:
this changes the indexes, so invalidates past storage contents, probably OK as you are likely only user, but will need to stop doing this as soon as there is more than one user

Reply:
Yup. I thought it best to get this cleanup done as soon as possible so that it is done before others use it.